app/views/jst/courses/roster/rosterUser.handlebars

```diff
@@ -2,7 +2,15 @@
   {{>avatar}}
 </td>
 <td>
-  <a href="{{html_url}}" class="roster_user_name">{{name}}</a>
+  <a href="{{html_url}}" class="roster_user_name">
+      {{! SFU MOD}}
+      {{#if canViewSisIdColumn}}
+          {{name}}（{{short_name}}）
```

Comment:
@GinnyGuo If a user does not have a `short_name` set, Canvas returns the value of the `name` field when `short_name` is called. This means that for those users, you'll be displaying something like "Graham Neil Ballantyne (Graham Neil Ballantyne)". Is that intended behaviour?

![screenshot 2016-04-25 09 58 07](https://cloud.githubusercontent.com/assets/146111/14793472/3792cd1a-0ad4-11e6-834c-8529ad52072d.png)